interfaces/Libraries.ts

```diff
@@ -161,6 +162,32 @@ const timm = (model: ModelData) =>
 
 model = timm.create_model("${model.modelId}", pretrained=True)`;
 
+const sklearn_with_filename = (model: ModelData, filename: string) =>  
+`from huggingface_hub import hf_hub_url, cached_download
+import joblib
+
+model = joblib.load(cached_download(
+	hf_hub_url("${model.modelId}", "${filename}")
+))`;
+
+const sklearn_no_joblib = (model: ModelData) =>  
+`from huggingface_hub import hf_hub_url, cached_download
+import joblib
+
+# No joblib file found!
+model = joblib.load(cached_download(
+	hf_hub_url("${model.modelId}", "name of file")
+))`;
+
+const sklearn = (model: ModelData) => {
+	const joblib_file = model.siblings?.find(file => file.rfilename.endsWith(".joblib"));
```

Comment:
In my mind we were enforcing a model filename of `sklearn_model.joblib` (similar to what we're doing for pt/tf/jax/etc)

Is there any downside to enforcing a fixed filename?

Reply:
No strong opinion on this, just trying to be a bit more flexible.

Let's go with the fixed filename as we do for pt/tf/jax :)

Reply:
yeah otherwise someone will have multiple .joblib files in a repo and we don't know which one to run in inference :)